docs/components/secnav/test.html

```diff
@@ -5,113 +5,113 @@
 
 {% block content %}
 <div id="vue-secNavDemo">
-  <nav id="hxTopNav">
-    <a class="hxTopNavLogo" href="#">
-      <img src="images/helix-logo.svg" alt="Helix Brand Logo" />
-    </a>
-    <div class="hxTopNavMenu hxTopNavOptionMenu">
-      <hx-disclosure aria-controls="topnav-menu-options">
-        <span>Select an option</span>
-        <hx-icon class="hxPrimary" type="angle-down"></hx-icon>
-      </hx-disclosure>
-      <hx-menu id="topnav-menu-options">
-        <section>
-          <hx-menuitem>Option Alpha</hx-menuitem>
-          <hx-menuitem>Option Beta Services</hx-menuitem>
-          <hx-menuitem>Option Gamma</hx-menuitem>
-        </section>
-      </hx-menu>
-    </div>
-    <div class="hxTopNavIconMenu">
-      <div>
-        <a href="#" @click="notify()">
-          <hx-icon type="bell"></hx-icon>
-          <p>Notifications</p>
+    <nav id="hxTopNav">
+        <a class="hxTopNavLogo" href="#">
+          <img src="images/helix-logo.svg" alt="Brand Logo" />
         </a>
-
-        <a href="#" class="selected">
-          <hx-icon type="ticketing"></hx-icon>
-          <p>Tickets</p>
-        </a>
-
-        <a href="#">
-          <hx-icon type="support"></hx-icon>
-          <p>Support</p>
-        </a>
-
-        <a href="#" class="hxDisabled" id="billing">
-          <hx-icon type="billing"></hx-icon>
-          <p>Billing</p>
-          <hx-tooltip for="billing" position="bottom-center">
-            You do not have access to this area. Contact an account admin
-            in your organization to request access to this item.
-          </hx-tooltip>
-        </a>
-
-        <a href="#">
-          <hx-icon type="account"></hx-icon>
-          <p>Account</p>
-        </a>
-      </div>
-      <div class="hxSpacer"></div>
-      <div class="hxTopNavMenu">
-        <hx-disclosure aria-controls="demo-user-menu">
-          <span>Jane User</span>
+        <div class="hxTopNavMenu hxTopNavOptionMenu">
+          <hx-disclosure aria-controls="topnav-menu-options" aria-expanded="true">
```

Comment:
Please remove the `aria-expanded` attribute, since this is the demo for Secondary Navigation.

Reply:
Changes have been done, please review.